lib/standard/collection/union_find.nit

```diff
@@ -33,10 +33,46 @@ import hash_collection
 # The methods `in_same_subset`, `to_partitions`, and their variations are offered instead.
 class DisjointSet[E]
 	super SimpleCollection[E]
+	super Cloneable
 
 	# The node in the hiearchical structure for each element
 	private var nodes = new HashMap[E, DisjointSetNode]
 
+	# Copy constructor
+	init from(other: DisjointSet[E])
```

Comment:
`from` is already used by some classes to be constructed from other classes like `Collection::from`. I see a risk of confusion here.

Maybe we should state once and for all the name of a copy constructor like `copy`?